Resources/Textures/_RMC14/Objects/Weapons/Guns/SmartGuns/GSH.rsi/meta.json

```diff
@@ -0,0 +1,47 @@
+﻿{
+  "version": 1,
+  "license": "CC-BY-SA-4.0",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/commit/3163b00, suiststorage sprites made by VictorJob",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "wielded-inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "wielded-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "mag-0"
+    },
+    {
+      "name": "base"
+    },
+    {
+      "name": "bolt-open"
+    },
+    {
+      "name": "equipped-BACKPACK",
+      "directions": 4
+    },
+    {
+      "name": "Icon"
```

Comment:
This needs to be lowercase, same for the corresponding filename

Reply:
fixed, also fixed the smart gun that had an "Icon" in both the meta and the sprite name